Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -960,6 +926,109 @@ namespace gdjs {
       );
     }
 
+    /**
+     * Tween a numeric object effect property.
+     * @param effectBehavior Only used by events can be set to null
+     * @param identifier Unique id to identify the tween
+     * @param toValue The targeted value
+     * @param effectName Effect name
+     * @param propertyName Property name
+     * @param easing Easing function identifier
+     * @param duration Duration in seconds
+     * @param destroyObjectWhenFinished Destroy this object when the tween ends
+     */
+    addNumberEffectPropertyTween(
+      effectBehavior: any,
+      identifier: string,
+      toValue: float,
+      effectName: string,
+      propertyName: string,
+      easing: string,
+      duration: float,
+      destroyObjectWhenFinished: boolean
+    ) {
+      const effect = this.owner.getRendererEffects()[effectName];
+      this._tweens.addSimpleTween(
+        identifier,
+        this.owner,
+        duration,
+        easing,
+        linearInterpolation,
+        effect ? effect.getDoubleParameter(propertyName) : 0,
```

Comment:
Should this error if there is no effect?

Reply:
Do you you think I should add an error log?
I'm not sure if it's necessary because it's a drop-down list in the UI (though it's missing for object effects)